File: math/leap-year-function.js
function leapYear(year){
  return year % 4 === 0 && (year % 100 !== 0 && year % 400 !== 0)
}

leapYear(1850) // false
leapYear(1565) // false
leapYear(1920) // true
leapYear(1924) // true
leapYear(1928) // true
leapYear(1940) // true
leapYear(2021) // false

File: algebra/leap-year-function.py
def leap_year(year):
	return year % 4 == 0 and year % 100 != 0 or year % 400 == 0

leap_year(1850) # False
leap_year(1565) # False
leap_year(1920) # True
leap_year(1924) # True
leap_year(1928) # True
leap_year(1940) # True
leap_year(2021) # False
leap_year(1900) # False
leap_year(2000) # True